interview-questions.md

```diff
@@ -8,30 +8,30 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is an incorporation of languages HTML and JavaScript for React. 
 
-Researched answer:
+Researched answer: JSX is short for JavaScript XML. It allows developers to write HTML-like codes within Javascript and is a syntax extension for JavaScript which is usually used with React. One example of a different syntax between HTML and JSX is the use of class and className. The attribute class/className is is used to define one or more CSS classes for an element. class is utilized for HTML and className is utilized for JSX. This is to avoid naming crashing. 
 
 2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: Yarn is a package manager that allows users/developers test their codes. When running the command yarn in terminal, the files that are modified includes package.json.
 
-Researched answer:
+Researched answer: Yarn is a package manager for JavaScript and Node.js projects. Yarn is used to manage projects, install packages, and perform any task that is related to maangement and development of the project. When running yarn on terminal, modification and creation of file(s) include node_modules,  yarn.lock, package.json, .pnp.js(optional because it does not rely on node_modules), .yarnrc (optional for only if you have a .yarnrc file), and yarn-error.log(optional for only if an error occurs during installation). 
 
 3. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: Props which is short for properties, is used to pass data which needs to be determined by the developer. For example, calling props from a function. Ex: { props.name } props is the argument to a component and .name is utilized to retrieve the name property of props. State values refers to useState (?). For example: [value, setValue]. value is the state variaable and setValue is the update to value when it's called upon. 
```

Comment:
Yup, `useState` is called a React hook. It hooks into the fundamental React functionality. State allows for a variable that is available to the whole component and a method that can update the value of that variable.